### PR TITLE
Uwa 378/invoicing address

### PR DESCRIPTION
**Pull request by bmoelans**

### CODE REVIEW

File: paths/subscriptions/invoicing_settings.yaml [hunk 1]
```diff
@@ -47,3 +47,26 @@ put:
       description: unauthorized
       schema:
         $ref: '#/definitions/Status'
+
+
+delete:
```
Comment (bmoelans) [inline paths/subscriptions/invoicing_settings.yaml]: not sure myself, do we want this, or do we want that we deleted if you do put with empty invoicing_address? Anyway, Comarch is missing this functionality right now.

Comment (Kjoep) [reply paths/subscriptions/invoicing_settings.yaml]: A delete makes no sense considering the other invoicing settings are part of the same entity, and it makes no sense to 'remove' those. (right now the only other setting is paper/email -- more could be added later)
The logical thing to do is to PUT it, but leave out the address.

File: definitions/subscriptions/invoicing_settings.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 type: object
 title: Invoice settings
+description: If there is no invoicing_address defined, the system will use the address of the customer.
```
Comment (freeduerinckx) [inline definitions/subscriptions/invoicing_settings.yaml]: I would rephrase is to: `if there is no contact information  (invoicing_address) defined `

File: paths/subscriptions/invoicing_settings.yaml (hunk 1 above)
Comment (JanJoris) [reply paths/subscriptions/invoicing_settings.yaml]: idd

Comment (JanJoris) [reply paths/subscriptions/invoicing_settings.yaml]: this does imply UWA will have to add your Customer address as there's no way to delete one's invoice address yet.

Comment (bmoelans) [reply paths/subscriptions/invoicing_settings.yaml]: for front-end this is not nice, you only want to show extra invoice_addresses if this calls gives something back. In that case I would e.g. suggest a temporary hack and put e.g. `_DELETED_` as company name so that we do not return it...
